crates/block-producer/src/runner.rs

```diff
@@ -158,6 +158,11 @@ async fn poll_loop(
 }
 
 pub fn run(config: Config, skip_config_check: bool) -> Result<()> {
+    // Enable metrics config.
+    #[cfg(feature = "tokio")]
```

Comment:
why checking tokio feature here? all the things should be hidden behind the gw_metrics crate

Reply:
I thought that way before. But that means I need to check gw_metrics feature when i add a new metric every time, which is weird as well..
Instead, I set tokio as an optional dependency and reexport tokio feature to upper level.
Or... I can put metrics utils into a new crate. So crates can always depend on metric-utils and optionall depend on metric-config. Sounds better?
Need to figure out it's feasible or not.

Reply:
As we discussed offline, we have two features: `default` and `metrics` on metrics crate.